lib/galaxy/webapps/galaxy/api/plugins.py

```diff
@@ -42,7 +39,7 @@ def index(self, trans, **kwargs):
             target_object = self.hda_manager.get_accessible(self.decode_id(dataset_id), trans.user)
         return registry.get_visualizations(trans, target_object=target_object, embeddable=embeddable)
 
-    @expose_api
+    @expose_api_anonymous_and_sessionless
```

Comment:
That seems fine, but can anon's save visualizations ? Is that properly disabled ?